content/syllabus-written-assignments/introduction/_index.md

```diff
@@ -0,0 +1,19 @@
+---
+content_type: topic
```

Comment:
This file doesn't seem to be referenced anywhere. That means nobody will ready it.

I'm not sure that it adds value anyway. It would need a bit of work before we can include it. To speed things up I'd recommend removing it from the  PR. Maybe make a separate issue about introducing learners to writing assignments. But I'm not sure if they need an introduction or not.